pkg/deploy/api/types.go

```diff
@@ -441,6 +441,20 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta
+	// Name of the deployment config for requesting a new deployment.
+	Name string
+	// DryRun if true should not persist the updated deployment config in storage.
+	DryRun bool
+	// Latest will update the deployment config with the latest images found in the cluster.
+	Latest bool
+	// Manual should always try to force a new deployment to run. This is meant to be used
+	// by all clients.
+	Manual bool
+}
```

Comment:
Yes.  This is forcing a generation / latestVersion change, which is why we want to explicitly state that in the godoc and have the name reflect the change.  manual doesn't mean anything and we try not to use it.